### DISCUSSION TITLE
HB - Search Theme and Url - Resolution

### DISCUSSION THREAD

**Discussion by Gordolin** [hbstack/.github#87] [Q&A] [answered]
Hi, i tried your new search Theme. Great Job, that was a thing that i have missed.

I have some sites on a local synology nas.
The sites can be called with i.e. https://mynas/site1, https://mynas/site2, ....
The baseURL in hugo.yaml is 'baseURL: https://mynas/site1' ....

in site-search is configured:
- https://mynas/site1/search.json
- https://mynas/site2/search.json 
....

That works as described, search results are shown as expected, but the link to the search results is wrong. 
The link is  https://mynas/site1/site1/docs/.... instead of https://mynas/site1/docs/....

With a baseURL without subfolder erything works as descibed.
Is there a way to fix that?

**Comment by razonyang** [comment, marked as answer]
Thank you for reporting, it's a bug.

Please upgrade the search and search-index module, and rebuild the sites to regenerate the index and then force refresh the browser on the search page, make sure there isn't caching issues.

```sh
hugo mod get -u github.com/hugomods/search
hugo mod get -u github.com/hugomods/search-index
```
